operators/test/e2e/test/elasticsearch/steps_deletion.go

```diff
@@ -54,5 +58,23 @@ func (b Builder) DeletionTestSteps(k *test.K8sClient) test.StepList {
 				return k.CheckPodCount(test.ESPodListOptions(b.Elasticsearch.Namespace, b.Elasticsearch.Name), 0)
 			}),
 		},
+		{
+			Name: "Remove leftover PVCs",
+			// TODO: remove when https://github.com/elastic/cloud-on-k8s/issues/1288 is fixed.
+			Test: func(t *testing.T) {
+				var pvcs corev1.PersistentVolumeClaimList
+				err := k.Client.List(&client.ListOptions{
+					Namespace: b.Elasticsearch.Namespace,
```

Comment:
Use ManagedNamespace?

Reply:
The ES resource is created with the ManagedNamespace, the test itself should only work from the ES resource (only input) with no dependency to test.Ctx() if not required IMO.